mohammad/JS/JS-dataFetchingAndDOM/add.js

```diff
@@ -0,0 +1,27 @@
+const url = "https://jsonplaceholder.typicode.com";
+
+function submit() {
+  const title = document.getElementById("title").value;
+  const body = document.getElementById("body").value;
+  const author = document.getElementById("author").value;
```

Comment:
This field seems not to be used.

Reply:
you are right
at first in my first implementation I need this field but in my last implementation I don't need this field and I forgot to delete it.